indra/newview/skins/default/xui/en/panel_main_inventory.xml

```diff
@@ -217,6 +217,7 @@
        tab_height="30"
        tab_position="top"
        tab_min_width="100"
+       use_tab_offset="true"
```

Comment:
arguably tabs are the ones that should be fixed to not require an additional offset.